de.dlr.sc.virsat.graphiti/src/de/dlr/sc/virsat/graphiti/diagram/BeanPropertyFactory.java

```diff
@@ -0,0 +1,89 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.graphiti.diagram;
```

Comment:
I only now notice that the factory is in the graphiti plugin.
Maybe we can move it to where the other bean factories are?
I dont see this as a graphiti thing.